packages/ui/hooks/useBorrowMinimum.ts

```diff
@@ -34,8 +34,8 @@ export const useBorrowMinimum = (asset: FuseAsset) => {
       minBorrowUSD: Number(utils.formatUnits(response.data, 18)) * usdPrice,
       minBorrowNative: response.data,
       minBorrowAsset: response.data
-        .mul(utils.parseUnits('1', asset.underlyingDecimals))
-        .div(asset.underlyingPrice),
+        .div(asset.underlyingPrice)
+        .mul(utils.parseUnits('1', asset.underlyingDecimals)),
```

Comment:
https://github.com/Midas-Protocol/monorepo/commit/ba7d5d0450a1dcea847ca6f2d79426408f669636

This change fixed the issue of the 0.35 showing as zero

![Screenshot 2022-09-13 at 13 22 52](https://user-images.githubusercontent.com/9201443/189893466-85985582-8638-4271-b903-1d5427b27661.png)